### PR TITLE
82-highlight-hashtags

### PR DESCRIPTION
**Pull request by MarshallARoss**
Closes on issue #82

## What it Does
* Add a function to highlight hashtags in tweets.
* Add example tweets to AWSTweet model that contains hashtags for the preview provider.

## How I Tested
* Run the application
* Go into your collections and look for a tweet with a hashtag 
* If the hashtag is at the beginning of a word the word will be highlighted twitter blue
* if the hashtag symbol is in the middle of a word the word will be the normal color. Just like Twitter does.

## Screenshot
Simulator Screen Shot - iPhone 13 Pro - 2022-10-12 at 17 42 48

### CODE REVIEW

**Comment by mikaelacaron** [comment]
@MarshallARoss Also be sure to squash your commits into a single commit, see [here](https://github.com/mikaelacaron/brain-marks#getting-started)

File: amplify/generated/models/AWSTweet.swift [hunk 1]
```diff
@@ -35,4 +35,19 @@ public struct AWSTweet: Model {
       self.category = category
       self.userVerified = userVerified
   }
-}
\ No newline at end of file
+}
+
+extension AWSTweet {
```
Comment (mikaelacaron) [inline amplify/generated/models/AWSTweet.swift]: Don't modify this file, it's generated, instead use `AWSTweet+Extension`

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
I haven't looked at your PR throughly yet, but I noticed this.
Also my other comment about squashing your commits into a single commit

**Comment by MarshallARoss** [comment]
Resubmitting as one commit and with example tweets moved to AWSTweet+Extension file

**Comment by mikaelacaron** [comment]
That's fine! But you don't have to close it, you can just update your branch and push and the PR will update

**Pull request closed without merging**